services/db/workspaceTableHandler.go

```diff
@@ -0,0 +1,104 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/rudderlabs/rudder-server/config"
+)
+
+var (
+	host, user, password, dbname string
+	port                         int
+)
+
+type WorkspaceDataT struct {
+	token      string
+	created_at string
+	parameters string
+}
+
+func loadOriginalDBConfig(originalDBname string) {
+	host = config.GetEnv("JOBS_DB_HOST", "localhost")
+	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
+	dbname = originalDBname
+	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
+	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
+
+}
+
+func loadDBConfig() {
+	host = config.GetEnv("JOBS_DB_HOST", "localhost")
+	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
+	dbname = config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
+	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
+	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
+
+}
+
+func createWorkspaceTable(dbHandle *sql.DB) {
+	sqlStatement := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS workspace (
+		token TEXT PRIMARY KEY,
+		created_at TIMESTAMP NOT NULL,
+		parameters JSONB);`)
+
+	_, err := dbHandle.Exec(sqlStatement)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func GetConnectionString() string {
```

Comment:
EnvValidator does everything that we do here. We should either use that or make envValidator use this.Let's not repeat the code.